zebra-chain/src/error.rs

```diff
@@ -0,0 +1,38 @@
+//! Errors that can occur inside any `zebra-chain` submodule.
+use thiserror::Error;
+
+/// Number of times a `diversify_hash` will try to obtain a diversified base point.
+pub const DIVERSIFY_HASH_TRIES: u8 = 5;
+
+/// Errors related to random bytes generation.
```

Comment:
I'd like a cryptographer to review these error names and descriptions to make sure they match the terminology in the spec (or standard cryptographic terminology).